ui/src/app.jsx

```diff
@@ -113,15 +114,27 @@ export default React.createClass({
   },
 
   messagePopupAuthorQuestions(query = {}) {
-    return <MessagePopup.QuestionsPage />;
+    return (
+      <QuestionsLoader>
```

Comment:
I'd prefer the more explicit form where you pass a render function in as a prop for something like this (since this form hides how the props get passed into the other components), but this works too.